{{cookiecutter.project_name}}/justfile

```diff
@@ -0,0 +1,196 @@
+# On Windows the bash shell that comes with Git for Windows should be used.
+# If it is not on path, give the path to the executable in the following line.
+#set windows-shell := ["C:/Program Files/Git/usr/bin/sh", "-cu"]
+
+# Load environment variables from config.public.mk or specified file
+set dotenv-load := true
+# set dotenv-filename := env_var_or_default("LINKML_ENVIRONMENT_FILENAME", "config.public.mk")
+set dotenv-filename := x'${LINKML_ENVIRONMENT_FILENAME:-config.public.mk}'
```

Comment:
Ah, you do import the make config here. Maybe my concern about two files is moot?

Reply:
Yes it reads from the same file as the make file. Only the tasks are defined again the just-way.